src/Caching/Impl/ModuleFactory.cs

```diff
@@ -31,22 +31,26 @@ namespace Microsoft.Python.Analysis.Caching {
     /// Constructs module from its persistent model.
     /// </summary>
     internal sealed class ModuleFactory {
+        /// <summary>For use in tests so missing members will assert.</summary>
+        internal static bool EnableMissingMemberAssertions { get; set; }
```

Comment:
It will be shared between tests

Reply:
Yes, that is the intent. Basically, assert if cache tests cannot restore a member. At runtime this is normal until dependencies are fully analyzed.